src/config.rs

```diff
@@ -1,8 +1,19 @@
 //! # Configuration
 //!
 //! This module holds the data structures for network configuration.
+use crate::block::Fingerprint;
 use log::{error, info};
 use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+
+/// Configuration struct for a single bot
+#[derive(Debug, Serialize, Deserialize, Clone, Default)]
+pub struct BotConfig {
```

Comment:
```suggestion
/// Configuration struct for a single bot
#[derive(Debug, Serialize, Deserialize, Clone, Default)]
#[allow(clippy::module_name_repetitions)]
pub struct BotConfig {
```

since `BotConfig` is fine imho in this context and the upcoming clippy lint github action will complain about this